internal/api/api.go

```diff
@@ -133,6 +133,10 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings) *fiber.App {
 	v1.Get("/device-config/:vin/dbc", deviceConfigController.GetDBCFile)
 	v1.Get("/device-config/:vin/urls", deviceConfigController.GetConfigURLs)
 
+	v1.Get("/device-config/:template_name/pids", deviceConfigController.GetPIDsByTemplate)
```

Comment:
remove other endpoints for /:vin/pids

Reply:
removed

Reply:
I just want to say that it makes me happy when people respond to my review comments.

Reply:
:)